axbot_node:
  ros__parameters:
    robot_ip: "192.168.0.250"
    robot_port: 8090
    secret: ""
    ws_url: "ws://192.168.0.250:8090/ws/v2/topics"
    reconnect_interval: 5.0
    device_info_rate: 0.1
    base_frame: "base_link"
    odom_frame: "odom"
    initial_control_mode: "remote"
    twist_timeout: 2.0
    
    # Add Lidar topics here
    subscribe_topics:
      - "/tracked_pose"
      - "/slam_state"
      - "/wheel_state"
      - "/planning_state"
      - "/scan_matched_points2"      # combined point cloud for SLAM
      - "/horizontal_laser_2d/matched"  # individual lidar

